ember-cli-build.js

```diff
@@ -54,6 +55,45 @@ module.exports = function () {
+        theme_color: '#fff',                                                   // Theme color user for example in Android's task switcher. `string`
+        appleStatusBarStyle: 'black-translucent',                              // Style for Apple status bar: 'black-translucent', 'default', 'black'. `string`
+        display: 'standalone',                                                 // Preferred display mode: 'fullscreen', 'standalone', 'minimal-ui' or 'browser'. `string`
+        orientation: 'any',                                                    // Default orientation: 'any', 'natural', 'portrait' or 'landscape'. `string`
+        scope: '/',                                                            // set of URLs that the browser considers within your app
+        start_url: '/',                                                        // Start URL when launching the application from a device. `string`
+        version: '1.0',                                                        // Your application's version string. `string`
+        logging: false,                                                        // Print logs to console? `boolean`
+        icons: {
+          // Platform Options:
+          // - offset - offset in percentage
+          // - background:
+          //   * false - use default
+          //   * true - force use default, e.g. set background for Android icons
+          //   * color - set background for the specified icons
+          //   * mask - apply mask in order to create circle icon (applied by default for firefox). `boolean`
+          //   * overlayGlow - apply glow effect after mask has been applied (applied by default for firefox). `boolean`
+          //   * overlayShadow - apply drop shadow after mask has been applied .`boolean`
+          //
+          android: true,              // Create Android homescreen icon. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+          appleIcon: true,            // Create Apple touch icons. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+          appleStartup: true,         // Create Apple startup images. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+          coast: false,               // Create Opera Coast icon. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+          favicons: true,             // Create regular favicons. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+          firefox: false,             // Create Firefox OS icons. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+          windows: true,              // Create Windows 8 tile icons. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+          yandex: false               // Create Yandex browser icon. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
```

Comment:
😟

Reply:
@artursmirnov Could you elaborate? I specifically asked for feedback on what to support, so if you're concerned about leaving something off, I'm happy to hear it :)

Reply:
@clekstro I think that your choice totally makes sense 👍 I'm using Yandex Browser myself on both mobile and desktop, but I think I'm pretty much unique on this in Travis audience. It just doesn't worth adding because of very low market share :)